apps/backend/src/resources/spec/fixtures/sale.json

```diff
@@ -0,0 +1,41 @@
+{
+    "id": "test",
+    "sale_date": "2022-10-12",
+    "journal_name": "SM J000001",
+    "ministry_alpha_identifier": "SM",
+    "total_amount": 50,
+    "distributions": [
+        {
+            "line_number": "00001",
```

Comment:
It looks like the tab width is extra wide here, but possibly this is intentional?